backend/inspirehep/records/marshmallow/literature/ui.py

```diff
@@ -150,6 +155,19 @@ def get_number_of_references(self, data):
         references = data.get("references")
         return self.get_len_or_missing(references)
 
+    def get_linked_book(self, data):
+        parent = get_parent_record(data)
+        if parent and "titles" in parent and "control_number" in parent:
+            endpoint = PidStoreBase.get_endpoint_from_pid_type(
+                PidStoreBase.get_pid_type_from_schema(data["$schema"])
+            )
+            endpoint_item = f"invenio_records_rest.{endpoint}_item"
+            ref = get_value(parent, "self.$ref") or url_for(
+                endpoint_item, pid_value=parent["control_number"], _external=True
+            )
+            return {**parent["titles"][0], "$ref": ref}
```

Comment:
what I was trying to say on the previous review, was to have `{**parent["titles"][0], "recod": {"$ref": ref}}` because this is how it's done all over the codebase, we wrap `$ref` properties in a `record` dict.

Reply:
But then it will break schemas: https://github.com/inspirehep/inspire-schemas/blob/3e91dc065426896b745f468495bf5bcdc8fbdff0/inspire_schemas/records/hep.yml#L1452-L1453

Reply:
I don't see why it should

Reply:
it's for `parent_record` and this is what it's creating here.